### PR TITLE
Create feature file for CCA Create Dip script

### PR DESCRIPTION
**Pull request by joel-simpson**
As part of the CCA project we are creating a Create DIP script. 
See https://github.com/artefactual-labs/archivematica-acceptance-tests/issues/29
This is a first draft of that script

### CODE REVIEW

**Comment by joel-simpson** [comment]
@jraddaoui would love your thoughts on this first draft of the feature file. 
I spoke to @jhsimpson briefly about this, he suggested that we follow the pattern described by the [configuration documentation](https://github.com/artefactual/automation-tools/tree/45400bf16d3e2a4646899d91393a41b72f737be4#configuration) for the automation tools. The key point I take from that is to call the create_dip.py from a shell script; and the shell script is what can include the environment parameters (ss_url etc.). 
Not sure if that is worth trying to describe in the feature file (we may want to create some basic documentation for this and could describe these details there?).

**Comment by jraddaoui** [comment]
Great @joel-simpson!

I'm following that pattern for the new script, there is more info in https://projects.artefactual.com/issues/11480#note-3. I'm not completely sure about its integration with this repository, but I'll take a deeper look at that along the week.

Thanks!

File: features/CCA/create_CCA_dip.feature [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Then the DIP Creation script will retrieve the AIP
+  And creates a new DIP named with the original Transfer name appended with “-DIP”
+  And creates a DIP-METS.xml file that describes the contents of the DIP
+  And creates an objects directory containing one zip container with all objects from the   original transfer
+  And ensures each object has it's original directory or file name in their original order
+  And updates the 'last modified date' of each file using the fslastmodified property from the AIP METS file (if available)
+  And includes a copy of the submissionDocumentation included in the original Transfer
+  And includes a copy of the AIP METS file generated during ingest
+  And stores the DIP (locally... where?)
```
Comment (sevein) [inline features/CCA/create_CCA_dip.feature]: @jraddaoui, Justin just mentioned to me that uploading the DIP to SS should be considered optional. It's ok for now to store it in a location in the filesystem given by the user.

**Comment by jraddaoui** [comment]
Moved to https://github.com/artefactual-labs/archivematica-acceptance-tests/pull/35 to use a branch from this repository.

**Pull request closed without merging**